db/seeds.rb

```diff
@@ -7,8 +7,35 @@
 #   Mayor.create(name: 'Emanuel', city: cities.first
 
 user = User.create(
-  username: username
+  username: "username",
   email: "user@email.com",
   password: "password",
   password_confirmation: "password"
 )
+
+bucket_list = BucketList.create(
+  title: 'Bucket List 1',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.